cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxGLSurfaceView.java

```diff
@@ -68,6 +68,16 @@ public void setSoftKeyboardShown(boolean softKeyboardShown) {
 
     private boolean mSoftKeyboardShown = false;
 
+    public boolean isMultipleTouchEnabled() {
+        return mMultipleTouchEnabled;
+    }
+
+    public void setMultipleTouchEnabled(boolean multipleTouchEnabled) {
+        this.mMultipleTouchEnabled = multipleTouchEnabled;
+    }
+
+    private boolean mMultipleTouchEnabled = true;
```

Comment:
it is better to define the variable at the beginning of the class as other variables